esrally/mechanic/launcher.py

```diff
@@ -281,11 +216,29 @@ def start(self, node_configurations=None):
         return c.nodes
 
     def stop(self, nodes):
-        # nothing to do here, externally provisioned clusters / nodes don't have any specific telemetry devices attached.
+        # nothing to do here, externally provisioned clusters / nodes don't have any specific telemetry devices
+        # attached.
         pass
 
 
-class InProcessLauncher:
+def _get_pid_from_file(filename):
+    with open(filename, "r") as f:
+        return int(f.read())
+
+
+def wait_for_pidfile(pidfilename, timeout=10):
```

Comment:
We have seen situations where the JVM was slow to startup due to fragmented memory on the system so I suggest to raise this timeout to 30 seconds or maybe even 60.